packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -200,9 +195,8 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
   title?: string;
 
   /**
-   * Method to provide the classnames to style the contextual menu. Default value is the getMenuClassnames func
-   * defined in ContextualMenu.classnames.
-   * @default getContextualMenuClassNames
+   * Method to provide the classnames to style the contextual menu.
```

Comment:
The comment re: `getMenuClassnames` as default was inaccurate. Looks like we haven't been using that since we started to shift towards `mergeStyles`.